src/components/common/button/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import { observer } from 'mobx-react';
+import React from 'react';
+
+import './button.scss';
+
+interface ButtonProps {
+  onPlay: () => void;
+  exiting: boolean;
+  loading: boolean;
+}
+
+@observer
+export class Button extends React.PureComponent<ButtonProps> {
+  public render() {
+    const { onPlay, exiting, loading } = this.props;
+
+    const buttonText = loading ? 'loading' : 'play';
```

Comment:
'text' should be a prop for this button - so the button display the given text unless it's loading, in which case it always just says loading